src/main/java/frc/robot/subsystems/gpm/Intake.java

```diff
@@ -0,0 +1,108 @@
+package frc.robot.subsystems.gpm;
+
+import com.revrobotics.CANSparkFlex;
+import com.revrobotics.CANSparkLowLevel;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.RobotBase;
+import edu.wpi.first.wpilibj.simulation.DIOSim;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.constants.IntakeConstants;
+
+
+public class Intake extends SubsystemBase {
+
+    public enum Mode {
+        INTAKE(IntakeConstants.INTAKE_POWER), DISABLED(0)
+        ;
```

Comment:
Put this ; on the previous line. You can put the 2 modes on separate lines if you want to, but you can choose.

Reply:
done